src/screens/TopicPageScreen/elements/TopicDomainHeader.js

```diff
@@ -48,7 +48,6 @@ const TopicDomainHeader = (props) => {
         <Shimmer height={10} width={normalize(60)} />
       ) : (
         isFollow &&
-        followType === 'signed' &&
         !hideSeeMember && (
           <Pressable onPress={handlePress} style={{backgroundColor: COLORS.transparent}}>
             <Text style={styles.seeMemberText(followType)} numberOfLines={1} ellipsizeMode="tail">
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of the `followType === 'signed'` condition might lead to unexpected behavior if there are other possible values for `followType`. If there are other possible values, and they require different handling, this change could introduce bugs. Please ensure that this is the intended behavior.
```diff
-        followType === 'signed' &&
+        // Ensure that removing this condition doesn't affect other possible values of followType
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->